src/Courses/TopicGrades/GradeInfoHeader.tsx

```diff
@@ -310,13 +310,15 @@ export const GradeInfoHeader: React.FC<GradeInfoHeaderProps> = ({
                 <h4>Statistics</h4>
                 Number of attempts: <strong>{info.attemptsCount}</strong><br />
                 Best overall score: <strong>{info.overallBestScore?.toPercentString()}</strong><br />
+                {/* Should check !moment(topic.endDate).isSame(topic.deadDate) for topic and overrides before showing this? */}
+                Best score (including Partial Credit): <strong>{info.partialCreditBestScore?.toPercentString()}</strong><br />
```

Comment:
We need to be careful, this is unclear what this difference is to the user

I think this should simply say:
`System Score`

As in the score you would have gotten from the system without professor intervention, I think adding more details makes it confusing

Reply:
Also it might have been unclear from the ticket but this is what I call the `Topic Grading Page`
We already had a grading page and it was the grids